specification/hdinsight/resource-manager/Microsoft.HDInsight/preview/2024-08-01-preview/cluster.json

```diff
@@ -2028,10 +2036,36 @@
           "x-ms-client-name": "password",
           "description": "The gateway settings user password.",
           "readOnly": true
+        },
+        "restAuthMsEntra.Users": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/MsEntraUserInfo",
+            "description": "Details of an Entra user for gateway access."
+          },
+          "description": "List of Entra users for gateway access."
         }
       },
       "description": "Gateway settings."
     },
+    "MsEntraUserInfo": {
```

Comment:
>Ms [](http://example.com/codeflow?start=5&length=2)

Is "Ms" prefix really required?

Reply:
Resolved in latest commit, Please check commit ID a485479.